types/clamscan/index.d.ts

```diff
@@ -38,14 +38,14 @@ declare class NodeClam {
     settings: {
```

Comment:
Please export settings to a separate interface and use that wherever you need. Example
```
interface NodeClamSettings interface NodeClamSettings {
    removeInfected: boolean;
    quarantineInfected: boolean;
    scanLog: string | null;
    debugMode: boolean;
    fileList: string | null;
    scanRecursively: boolean;
    clamscan: {
        path: string;
        scanArchives: boolean;
        db: string;
        active: boolean;
    };
    clamdscan: {
        socket: boolean;
        host: boolean;
        port: boolean;
        timeout: number;
        localFallback: boolean;
        path: string;
        configFile: string | null;
        multiscan: boolean;
        reloadDb: boolean;
        active: boolean;
        bypassRest: boolean;
    };
    preference: string;
}
```